test/GPv2AllowListAuthenticator.test.ts

```diff
@@ -3,16 +3,16 @@ import { Contract } from "ethers";
 import { ethers, waffle } from "hardhat";
 
 describe("GPv2AllowListAuthentication", () => {
-  const [owner, nonOwner, solver] = waffle.provider.getWallets();
+  const [deployer, owner, nonOwner, solver] = waffle.provider.getWallets();
   let authenticator: Contract;
 
   beforeEach(async () => {
     const GPv2AllowListAuthentication = await ethers.getContractFactory(
       "GPv2AllowListAuthentication",
+      deployer,
     );
 
-    // Owner will default to admin account declared above.
-    authenticator = await GPv2AllowListAuthentication.deploy();
+    authenticator = await GPv2AllowListAuthentication.deploy(owner.address);
   });
 
   describe("constructor", () => {
```

Comment:
```suggestion
  describe("constructor", () => {
    it("should set the owner", async () => {
```

**note**: This suggestion is for the following line, GH doesn't let me add a comment on that line, however.